scripts/cronJobs.js

```diff
@@ -1,15 +1,117 @@
 const cron = require('node-cron');
-const userRecs = require('./simulateUserRecs.js');
-const genreRecs = require('./calculateGenreRecs');
+const request = require('request');
+const genreRecs = require('./calculateGenreRecs.js');
 
-const rand = () => Math.floor(Math.random() * 150) + 50;
+const genreArr = [
+  'action',
+  'animation',
+  'comedy',
+  'documentary',
+  'drama',
+  'family',
+  'fantasy',
+  'international',
+  'horror',
+  'musical',
+  'mystery',
+  'romance',
+  'sci_fi',
+  'thriller',
+  'western',
+];
 
-// simultate GET requests for individual user recommendations
-for (let i = 0; i < 10; i += 1) {
-  cron.schedule('* * * * *', () => {
-    setInterval(userRecs, rand());
+const processSessionData = () => {
+  const events = [];
+  for (let i = 0; i < 5; i += 1) {
+    events.push({
+      movie: {
+        id: Math.floor(Math.random() * 300000),
+      },
+      progress: Math.floor(Math.random() * 100) / 100,
+      timestamp: new Date(),
+    });
+  }
+  const sessionData = {
+    userId: Math.floor(Math.random() * 1000000),
+    events,
+  };
+  const options = {
+    url: 'http://localhost:3000/tetraflix/sessionData',
+    headers: {
```

Comment:
Is this necessary? In other words, is your server actually checking for this information?

Reply:
I think it is necessary to specify the content-type so that my server knows it is receiving a JSON object.